recipes-omnect/omnect-health-check/omnect-health-check.bb

```diff
@@ -0,0 +1,56 @@
+SUMMARY = "omnect Health Check"
+DESCRIPTION = "Provide on-device files for run-time health checks."
+LICENSE = "MIT"
+LIC_FILES_CHKSUM = "\
+    file://${COMMON_LICENSE_DIR}/MIT;md5=0835ade698e0bcf8506ecda2f7b4f302 \
+"
+
+SRC_URI = "\
+	file://healthchecklib.sh \
+	file://omnect_health__coredumps.sh \
+	file://omnect_health__services.sh \
+	file://omnect_health__system_running.sh \
+	file://omnect_health__timesync.sh \
+	file://omnect_health_check.sh \
+	file://omnect_health_checks.json \
+	file://omnect_service_log.sh \
+	file://omnect_service_log.tmpfilesd \
+	file://omnect_service_log_analysis.json \
+"
+
+RDEPENDS:${PN} = "bash"
+
+do_install() {
+    # tpmfiles.d
+    install -d ${D}${libdir}/tmpfiles.d
+
+    # create tmpfiles.d entry to (re)create dir + permissions
+    install -m 0644 -D ${WORKDIR}/omnect_service_log.tmpfilesd     ${D}${libdir}/tmpfiles.d/omnect_service_log.conf
```

Comment:
imho the hole recipe should be part of the omnect-device-service recipe and thus
the content of omnect_service_log.tmpfilesd should be part of https://github.com/omnect/meta-omnect/blob/main/recipes-omnect/omnect-device-service/omnect-device-service/omnect-device-service.tmpfilesd or this file should be renamed to omnect-health-check.tmpfilesd

Reply:
if we want to see health check as part of omnect-device-service (on layer context) I can do that.
@JanZachmann what do you think?